packages/ui/src/common/components/BadgeViolet.ts

```diff
@@ -22,6 +24,20 @@ export const BadgeViolet = styled.span<BadgeVioletProps>`
   text-transform: uppercase;
   ${Overflow.Dots}
 
+  ${({ separated }) =>
+    separated &&
+    css`
+      margin: ${spacing(0, 0.5)};
+
+      &:last-child {
+        margin-right: 0;
+      }
+
+      &:first-child {
+        margin-left: 0;
+      }
+    `};
```

Comment:
I think that I would go with adding dedicated parent and put CSS rules there. My thinking here is that we don't necessarily need to add a property for that.

Something like below:

```tsx
const BadgeRow = styled(Row)`
  ${BadgeViolet} {
    margin: ...
  }
  // or using inline grid maybe? 
`
```